ui/config/proxy.ts

```diff
@@ -5,24 +5,29 @@
  * For details, please see
  * https://pro.ant.design/docs/deploy
  */
+const { PROXY_PORT } = process.env
+const DEV_TAEGET = `http://localhost:${PROXY_PORT}`
+const TEST_TAEGET = `http://localhost:${PROXY_PORT}`
+const PRE_TAEGET = `http://localhost:${PROXY_PORT}`
+
 export default {
   dev: {
-    '/api/': {
-      target: 'https://preview.pro.ant.design',
+    '/api': {
+      target: DEV_TAEGET,
       changeOrigin: true,
       pathRewrite: { '^': '' },
```

Comment:
See this doc: https://umijs.org/config#proxy

Reply:
have fixed